genon/preprocessor/facade/convert_processor.py

```
@@ -2941,118 +3128,174 @@ async def _process_request(self, request: Request, file_path: str, **kwargs: dic
+            build_docling_document,
+            build_tabular_vectors,
+        )
+        if self._xlsx_cfg["processing_mode"] == "tabular":
+            _log.info(f"[DocumentProcessor] xlsx tabular 직접 처리: {file_path}")
+            vectors = build_tabular_vectors(
+                file_path,
+                header_row=self._xlsx_cfg["header_row"],
+                multi_table=self._xlsx_cfg["multi_table"],
+            )
+            if not vectors:
+                raise GenosServiceException("1", f"chunk length is 0")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)